src/transformer/events/mod_assign/assignment_submitted.php

```diff
@@ -28,14 +28,15 @@ function assignment_submitted(array $config, \stdClass $event) {
     $assignment = $repo->read_record_by_id('assign', $assignmentsubmission->assignment);
     $lang = utils\get_course_lang($course);
 
+    $verb = utils\get_verb($config, $lang, 'submit');
```

Comment:
this will fail
In the `get_verb` function the case is `submitted`

Reply:
good spot